.github/workflows/main.yml

```
@@ -1236,3 +1194,117 @@ jobs:
       run: python -m pip install -e interfaces/sourcegen
     - name: Test YAML output generation
       run: sourcegen --api=yaml --output=build/yaml -v
+
+  matlab:
+    name: MATLAB on ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-22.04, ubuntu-24.04, windows-2025, macos-14]
+      fail-fast: false
+    env:
+      CANTERA_ROOT: ${{ github.workspace }}
+      CANTERA_DATA: ${{ github.workspace }}/data
+    runs-on: ${{ matrix.os }}
+    needs: [ubuntu-multiple-pythons, macos-multiple-pythons, windows]
+    timeout-minutes: 60
+    steps:
+    - uses: actions/checkout@v5
+      name: Checkout the repository
+      with:
+        persist-credentials: false
+    - name: Set Python version
+      # Python version does not matter as it is only used to run sourcegen
+      uses: actions/setup-python@v6
+      with:
+        python-version: "3.12"
+    - name: Install Brew dependencies (macOS)
+      run: |
+        echo "Running on `sysctl -n machdep.cpu.brand_string`"
+        brew install --display-times libomp hdf5
+      if: runner.os == 'macOS'
+    - name: Install Apt dependencies (Ubuntu)
+      run: |
+        echo "Running on `cat /proc/cpuinfo | grep 'model name' | head -n 1 | cut -d ':' -f 2`"
+        sudo apt update
+        sudo apt install libhdf5-dev libfmt-dev libopenblas0-openmp
+      if: runner.os == 'Linux'
+    - name: Install Python dependencies
+      # Install Python dependencies, which are used by 'dotnet build'
```

I assume this should be:
```suggestion
      # Install Python dependencies, which are used by sourcegen
```